CODE_OF_CONDUCT.md

```diff
@@ -7,7 +7,7 @@ In the interest of fostering an open and welcoming environment, we as contributo
 ## Our Standards
 
 Examples of behavior that contributes to creating a positive environment include:
-
+TRIDIV_SWAIN
```

Comment:
Thank you for your contribution but we're currently not accepting any more names added to the list of positive behaviours.

Reply:
😄